src/solvers/dgsem_tree/container_viscous_2d.jl

```diff
@@ -1,12 +1,7 @@
 mutable struct ViscousContainer2D{uEltype <: Real}
     u_transformed::Array{uEltype, 4}
-    # Using an outer fixed-size datastructure leads to nasty implementations,
-    # see https://github.com/trixi-framework/Trixi.jl/pull/1629#discussion_r1355293953.
-    # Also: This does not result in speed up compared to using tuples for the internal 
-    # datastructures, see 
-    # https://github.com/trixi-framework/Trixi.jl/pull/1629#discussion_r1363352188.
-    gradients::Vector{Array{uEltype, 4}}
-    flux_viscous::Vector{Array{uEltype, 4}}
+    gradients::Tuple{Array{uEltype, 4}, Array{uEltype, 4}}
+    flux_viscous::Tuple{Array{uEltype, 4}, Array{uEltype, 4}}
```

Comment:
Maybe

```suggestion
    gradients::NTuple{2, Array{uEltype, 4}}
    flux_viscous::NTuple{2, Array{uEltype, 4}}
```

to make it easier to read?